packages/main-ui/src/pages/components/congratulation_popup.rs

```diff
@@ -25,9 +24,7 @@ pub fn CongratulationPopup(
                 }
 
                 div { class: "leading-[24px] text-[16px] font-regular text-white flex flex-row items-center justify-center text-center tracking-[0.005em]",
-                    pre {
-                        "{tr.congratulation}"
-                    }
+                    pre { "{tr.congratulation}" }
```

Comment:
don't use pre. it use different font family due to code rendering.
Instead of it,

```suggestion
                    p {  class: "white-space-pre-line", "{tr.congratulation}" }
```